src/failed-spec-parser.js

```diff
@@ -1,13 +1,15 @@
-const FAILED_LINE = /at \[object Object\]\.<anonymous> \((.*)\)/g;
+const FAILED_LINES = /at \[object Object\]\.<anonymous> \((.*?):.*\)/g;
```

Comment:
We should define this as a non-constant within the body of the function, since `exec` modifies the state of the regex it is invoked on.